spec/features/form/form_navigation_spec.rb

```diff
@@ -89,4 +110,57 @@
       end
     end
   end
+
+  describe "Editing a log" do
```

Comment:
This should maybe a context instead of a describe, something like `context "When a user is editing a log" do`